### PR TITLE
Update Kokoro config to run on GitHub

### PR DESCRIPTION
**Pull request by omarismail94**

### CODE REVIEW

File: kokoro/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Kokoro Infrastructure
```
Comment (omarismail94) [inline kokoro/README.md:1]: Used same wording as here:

https://github.com/protocolbuffers/protobuf/tree/main/kokoro

File: kokoro/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+----------------------
+
+The files in this directory serve as plumbing for running tests under
+Kokoro, our internal CI.
+
+We have shared this part of our CI configuration in hopes that it is
```
Comment (bashir2) [inline kokoro/README.md:7]: I would suggest to drop this whole paragraph and replace it with:
"If there are any changes required to these config files, please file an issue."

**Review by bashir2 — COMMENTED** [review]
Thanks for fixing the CI.

File: kokoro/README.md (hunk 2 above)
Comment (omarismail94) [reply kokoro/README.md]: fixed

**Review by bashir2 — APPROVED** [review]

**Pull request merged**